Makefile.install

```diff
@@ -115,9 +115,18 @@ endif
 install-merlin:
 	$(INSTALLSH) $(FULLCOQLIB) $(wildcard $(INSTALLCMX:.cmx=.cmt) $(INSTALLCMI:.cmi=.cmti) $(MLIFILES) $(MLFILES) $(MERLINFILES))
 
+#NB: some files don't produce native files (eg Ltac2 files) as they
+#don't have any Coq definitions. Makefile can't predict that so we use || true
+#vos build is bugged in -quick mode, see #11195
 install-library:
 	$(MKDIR) $(FULLCOQLIB)
-	$(INSTALLSH) $(FULLCOQLIB) $(LIBFILES)
+	$(INSTALLSH) $(FULLCOQLIB) $(ALLVO:.$(VO)=.vo)
+	$(INSTALLSH) $(FULLCOQLIB) $(ALLVO:.$(VO)=.vos) || true
```

Comment:
I think if you use `$(wildcard ...)` rather than `|| true`, the build log won't be polluted with spurious "could not find ..." or whatever error messages.  (On the other hand, there won't be any indication of missing .vos files, which may or may not be worse...)